tools/release/generate_release_notes.rb

```diff
@@ -48,8 +48,8 @@
 coming_tag_index += 1 if coming_tag_index
 release_notes_entry_index = coming_tag_index || release_notes.find_index {|line| line.match(/^\[\[logstash/) }
 
-report << "[[logstash-#{current_release_dashes}]]" unless release_notes.any? { |line| line.match(/^\[\[logstash-#{current_release_dashes}/) }
-report << "=== Logstash #{current_release} Release Notes\n" unless release_notes.any? { |line| line.match(/^=== Logstash #{current_release}/)}
+report << "[[logstash-#{current_release_dashes}]]" unless release_notes.any? { |line| line&.match(/^\[\[logstash-#{current_release_dashes}/) }
```

Comment:
while I don't doubt that this fixes an issue where `release_notes` has a `nil` entry, I am baffled as to how `release_notes` could possibly _have_ a `nil` entry, as it is composed from the result of `String#split`, _possibly_ with a string inserted.